core/src/main/scala/org/bitcoins/core/protocol/dlc/execution/DLCExecutor.scala

```diff
@@ -136,6 +136,9 @@ object DLCExecutor {
       fundingTx: Transaction,
       fundOutputIndex: Int
   ): ExecutedDLCOutcome = {
+    require(
+      DLCUtil.checkOracleSignaturesAgainstContract(contractInfo, oracleSigs),
+      s"Incorrect oracle signatures and contract combination")
```

Comment:
I think this is fine (the alternative being checking when we receive the attestments, i.e. in the wallet) but just wanted to make note again that this reduces the capabilities of the library meaning that if anyone ever ends up in a weird state where they have enough of the oracle attestments to claim their funds but don't have all of them they'll need a custom build in order to claim their funds

Reply:
I don't think this is a huge concern. The `DLCAttestmentTLV` should be the way people deal with signatures, not the actual raw `SchnorrDigitalSignature`s